### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by aryan-f** [webstah/self-supervised-bss-via-multi-encoder-ae#2]
Hi. 
Could you please share your `requirements.txt` file? I've been trying several versions of `pytorch-lightning` and none of them seem to work with the code.
Thanks.

**Comment by mbwebster** [comment]
@aryan-f Sure, I'll take a look after work and get everything to work with the latest(ish) version of lightning and PyTorch, and then add a requirements.txt file.

Also, was the error within a notebook or with the trainer.py file? I might know what the problem is if it's the former.

**Comment by aryan-f** [comment]
Thank you so much. 

I first had a Memory Error which I figured must have something to do with data loader workers since I was running it on Windows so I disabled them. Now there’s a new problem with lightning trainer. No matter which version of the library I use, there seems to be some breaking changes compared to the version you used. Latest version that I tried was 1.10 iirc which didn’t support gradient clipping with manual training.

**Comment by mbwebster** [comment]
Oh okay, manual training isn't necessary for that experiment... just a holdover from earlier versions. I have updated the code and made the requirements.txt file. I am just doing another training run to make sure everything is behaving properly with hopefully what is an improved configuration, and then I'll push the changes.

I'll also upload model weights somewhere when I get a chance.

**Comment by mbwebster** [comment]
Okay. I'll try to upload the model weights later, but for now, I have added the requirements.txt file and made some updates to the triangle and circles experiment.
